.github/workflows/ci_docs.yml

```diff
@@ -72,10 +72,11 @@ jobs:
             channels:
               - conda-forge
               - nodefaults
+              - conda-forge/label/python_rc
           cache-downloads: true
           cache-environment: true
           create-args: >-
-            python=3.11
+            python=3.12
```

Comment:
Getting a solver error at fc5c445cee5f26ff3874c3a7a15102308ae7e4b7 in https://github.com/GenericMappingTools/pygmt/actions/runs/6385228582/job/17329506537#step:3:54:
```bash
  error    libmamba Could not solve for environment specs
      The following package could not be installed
      └─ python 3.12**  is not installable because it requires
         └─ _python_rc, which does not exist (perhaps a missing channel).
```
Seems like [`python-3.12.0rc3`](https://anaconda.org/conda-forge/python/files?version=3.12.0rc3) is in the main conda-forge channel, but it depends on `_python_rc` which is in the `conda-forge/label/python_rc` channel. 

Adding this `_python_rc` channel temporarily in 6bded071077cec408668f1296e934961390d03f6 to see if it works (and figure out what dependencies need to be updated on conda-forge), but should remove it before merging.

Reply:
The `conda-forge/label/python_rc` channel is no longer needed, see https://anaconda.org/conda-forge/python/files?version=3.12.0.